From 4c1e59f7d601f82bdad68bd86227d66b74dd09fe Mon Sep 17 00:00:00 2001
From: Adrian Alonso <aalonso@freescale.com>
Date: Tue, 18 Aug 2015 11:18:14 -0500
Subject: [PATCH 0239/5242] MLK-11380: arm: imx: dts: imx7d-sdb: add
 iomuxc_lspr pad settings

commit  eba7072d028af9c1d8995b10e2378cd0a7f65ffe from
https://source.codeaurora.org/external/imx/linux-imx.git

- Add additional pad settiongs for iomuxc_lspr controller
- Move pinctrl_pwm1 to iomuxc_lspr domain

Signed-off-by: Adrian Alonso <aalonso@freescale.com>

Fixed for upstream LPSR renaming in 4.14.
Dropped wdog_b change because upstream already handles this nicely.

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index b4ec97c..02fe35c 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -401,7 +401,14 @@
 	imx7d-sdb {
 		pinctrl_hog_2: hoggrp-2 {
 			fsl,pins = <
-				MX7D_PAD_LPSR_GPIO1_IO05__GPIO1_IO5	0x14
+				MX7D_PAD_LPSR_GPIO1_IO05__GPIO1_IO5	  0x14
+				MX7D_PAD_LPSR_GPIO1_IO07__GPIO1_IO7    0x59
+			>;
+		};
+
+		pinctrl_pwm1: pwm1grp {
+			fsl,pins = <
+				MX7D_PAD_LPSR_GPIO1_IO01__PWM1_OUT	  0x110b0
 			>;
 		};
 	};
-- 
1.7.9.5

